lib/game_interface.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'gamestate'
+
+class GameInterface
+
+  def initialize
+    castle_data = load_file
+    @game = Gamestate.new(castle_data)
+    @game.play
+  end
```

Comment:
It's a bit better style to not do anything in an initializer except set up the object. This object doesn't have state, so I wouldn't give it an initializer at all. I'd give it a method `play`. A pragmatist would make it static, a purist would make it an instance method.

Why this advice? Partly because `ClassName.new` is more like a noun phrase; you expect it to return something, not do something. Also because, when a class is more complicated, it might both have interesting state and also do things. In that case it's easier to use and test the class if setting up the state is separate from invoking the behavior.